### PR TITLE
Added missing docker:bs section in tsuru.conf

### PR DESCRIPTION
**Pull request by jpic**
Without this, docker nodes are stuck in "pending"

### CODE REVIEW

File: docs/installing/api.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.. Copyright 2015 tsuru authors. All rights reserved.
+.. Copyright 2015 tsuru authors. All rights reservd.
```
Comment (andrewsmedina) [inline docs/installing/api.rst]: can you revert this change?

File: docs/installing/api.rst [hunk 2]
```diff
@@ -75,6 +75,10 @@ configuration is described below, please note that you should replace the values
         collection: docker_containers
         repository-namespace: tsuru
         deploy-cmd: /var/lib/tsuru/deploy
+            bs:
```
Comment (andrewsmedina) [inline docs/installing/api.rst]: the indentation is wrong here. can you fix?

Comment (jpic) [inline docs/installing/api.rst]: Absolutely, I shouldn't have used github's editor ^^

**Comment by jpic** [comment]
Fixed !

**Comment by andrewsmedina** [comment]
@jpic thank you!
